Jenkinsfile

```diff
@@ -19,7 +19,7 @@ pipeline {
         }
         stage("Integration Tests") {
             steps {
-                sh "./gradlew integrationTest"
+                sh "./gradlew solrIntegrationTest"
```

Comment:
can you flip that around, back to `integrationTest` please ?

fixed by inverting the dependency: `solrIntegrationTest.dependsOn integrationTest`

Reply:
Won't it start up the stack twice, once without solr, then with?

Reply:
> can you flip that around, back to `integrationTest` please ?
> 
> fixed by inverting the dependency: `solrIntegrationTest.dependsOn integrationTest`

Not sure how you see this exactly, elaborate please or commit the change.

Reply:
I have pushed some changes for this, although I am not sure this is what you had in mind.

Feels a bit empty: I basically renamed solrIntegrationTest into integrationTest and made sure it's required by the solr compose rather than by default compose.